great_expectations/execution_engine/sqlalchemy_execution_engine.py

```diff
@@ -449,6 +440,128 @@ class MetricDomainTypes.
                     "SqlAlchemyExecutionEngine only supports the great_expectations condition_parser."
                 )
 
+        if "column" in domain_kwargs:
+            return selectable
+
+        if "column_A" in domain_kwargs and "column_B" in domain_kwargs:
+            if "ignore_row_if" in domain_kwargs:
+                if self.active_batch_data.use_quoted_name:
+                    # If case matters...
+                    # noinspection PyPep8Naming
+                    column_A_name = quoted_name(domain_kwargs["column_A"])
+                    # noinspection PyPep8Naming
+                    column_B_name = quoted_name(domain_kwargs["column_B"])
+                else:
+                    # noinspection PyPep8Naming
+                    column_A_name = domain_kwargs["column_A"]
+                    # noinspection PyPep8Naming
+                    column_B_name = domain_kwargs["column_B"]
+
+                ignore_row_if = domain_kwargs["ignore_row_if"]
+                if ignore_row_if == "both_values_are_missing":
+                    selectable = (
+                        sa.select([sa.text("*")])
+                        .select_from(selectable)
+                        .where(
+                            sa.and_(
+                                sa.column(column_A_name) != None,
+                                sa.column(column_A_name) != None,
+                            )
+                        )
+                    )
+                elif ignore_row_if == "either_value_is_missing":
+                    selectable = (
+                        sa.select([sa.text("*")])
+                        .select_from(selectable)
+                        .where(
+                            sa.or_(
+                                sa.column(column_A_name) != None,
+                                sa.column(column_A_name) != None,
+                            )
```

Comment:
I think this logic is wrong; it seems to be saying "keep if either value is present", which means if one value is missing and one present, the row is NOT ignored (backwards from the user requested behavior)

Reply:
@jcampbell You are right.  I fixed the typo in the second column name and also corrected the logic.  Thanks for catching this.